cli/bpmetadata/path.go

```diff
@@ -0,0 +1,85 @@
+package bpmetadata
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+const (
+	regexEx  = ".*/(examples/.*)"
```

Comment:
nit: for readability
```suggestion
	regexExamples  = ".*/(examples/.*)"
```

Reply:
done